crates/accelerate/src/circuit_library/pauli_evolution.rs

```diff
@@ -205,9 +206,9 @@ fn multi_qubit_evolution(
 /// Args:
 ///     num_qubits: The number of qubits in the Hamiltonian.
 ///     sparse_paulis: The Paulis to implement. Given in a sparse-list format with elements
-///         ``(pauli_string, qubit_indices, coefficient)``. An element of the form
-///         ``("IXYZ", [0,1,2,3], 0.2)``, for example, is interpreted in terms of qubit indices as
-///          I_q0 X_q1 Y_q2 Z_q3 and will use a RZ rotation angle of 0.4.
+///         ``(pauli_string, qubit_indices, rz_rotation_angle)``. An element of the form
+///         ``("XIYZ", [0,1,2,3], 2)``, for example, is interpreted in terms of qubit indices as
+///          I_q0 X_q1 Y_q2 Z_q3 and will use a RZ rotation angle of 2.
```

Comment:
Given that you reordered the pauli string abovem shouldn't it be:

```suggestion
///         X_q0 I_q1 Y_q2 Z_q3 and will use a RZ rotation angle of 2.
```

Reply:
Whoops, yes -- good catch!

Reply:
Also the circuit doesn't match that string -- both fixed in 55a5357!